src/make-unbuffered-no-cache-request.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+import superagent from "superagent";
+import type {SuperAgentRequest} from "superagent";
+import {makeAgent} from "./make-agent.js";
+import type {RenderGatewayOptions} from "./types.js";
+import {makeShouldRetry} from "./make-should-retry.js";
+import type {Logger} from "./shared/index.js";
+
+/**
+ * Make a request for a given URL without buffering or caching.
+ *
+ * This is not intended for direct use. Use makeRequest.
+ *
+ * @param {RenderGatewayOptions} options The options used to start the gateway.
+ * @param {string} url The URL to be requested.
+ * @param {Logger} logger The logger to use.
+ * @returns {SuperAgentRequest} A superagent request for the URL.
+ */
+export const makeUnbufferedNoCacheRequest = (
+    options: RenderGatewayOptions,
+    url: string,
+    logger: Logger,
+): SuperAgentRequest => {
+    const {name: gatewayName, requests: requestOptions} = options;
+
+    // Get an agent.
+    const agent = makeAgent(requestOptions?.keepAlive);
```

Comment:
The RRS makes an agent once and then shares it across all requests. We'll likely need to do that too. I'll work on that once the pieces are in place.